modules/graph_ql/main.tf

```diff
@@ -1,4 +1,133 @@
-data "aws_iam_policy_document" "graphql_task_execution_policy_document" {
+# Local Variables
+locals {
+  graphql_port = 5000
+}
+
+resource "random_id" "lb_name" {
+  byte_length = 4
+}
+
+resource "aws_security_group" "gql_security_group" {
+  name        = "${var.prefix}-gql"
+  description = "Allow inbound communication on container port."
+  vpc_id      = var.vpc_id
+
+  ingress {
+    description      = "Container port communication."
+    from_port        = local.graphql_port
+    to_port          = local.graphql_port
+    protocol         = "tcp"
+    cidr_blocks      = ["0.0.0.0/0"]
```

Comment:
We may want to put a card on the backlog for security research that will
- Check with security on allowing all ingress on a port from anywhere (we may have to limit this to our subnets) and provide an implementation plan if we have to limit it.
- Check with security to see if we need to move HTTP to HTTPS and what those implementation details may look like if we need to. (My hope is since it is within the private VPC we can keep it as HTTP)

Reply:
Found that I had to open traffic to allow the `aws_ecs_service` to retrieve the image. Created separate security group `gql_task_security_group`.